src/test/java/org/mifos/integrationtest/cucumber/stepdef/BillPayStepDef.java

```diff
@@ -197,8 +197,8 @@ public void iShouldBeAbleToExtractResponseBodyFromCallbackForBillPay() {
                 }
                 if(rootNode != null && rootNode.has("billId") && rootNode.get("billId").asText().equals("001")) {
                     String requestId = null;
-                    if (rootNode.has("requestId")) {
-                        requestId = rootNode.get("requestId").asText();
+                    if (rootNode.has("reason")) {
+                        requestId = rootNode.get("reason").asText();
```

Comment:
can you assert the reason with expected reason message?

Reply:
okay

Reply:
rename the variable to reason / message